packager/media/event/muxer_listener_internal.cc

```diff
@@ -122,6 +122,23 @@ void AddAudioInfo(const AudioStreamInfo* audio_stream_info,
     }
     audio_info->mutable_codec_specific_data()->set_ec3_channel_map(
         ec3_channel_map);
+    uint32_t ec3_channel_mpeg_value;
+    if (!CalculateEC3ChannelMPEGValue(codec_config, ec3_channel_mpeg_value)) {
+        LOG(ERROR) << "Failed to calculate EC3 channel configuration \
+          descriptor value with MPEG scheme.";
```

Comment:
Avoid using line break, which may not be properly rendered, use:
 
       LOG(ERROR) << "Failed to calculate EC3 channel configuration "
                      "descriptor value with MPEG scheme.";

Reply:
I will update based on your suggestion.